docs/DeveloperGuide.adoc

```diff
@@ -334,26 +434,81 @@ Use case ends.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The one or more given parameters are invalid.
++
+[none]
+** 3a1. Application shows an error message.
++
+Use case resumes at step 2.
+
+[discrete]
+=== Use case: Take notes for a patient
+
+*MSS*
+
+1.  User requests to open note GUI for a specific patient
+2.  User edit the patient’s notes
```

Comment:
The response of the system could be more explicit, eg "2. Application opens note GUI for specified patient"